src/pages/connect.tsx

```diff
@@ -168,7 +164,6 @@ const ConnectLanding: ThirdwebNextPage = () => {
             ctaLink="/dashboard/connect/playground"
             contactUsTitle="Book Demo"
             gradient="linear(to-r, #4490FF, #4490FF)"
-            lottie={connectLottie}
```

Comment:
we removed these, did we also delete the files? if not let's do that if they are no longer used